stdlib/database/h2/src/test/java/org/ballerinalang/stdlib/utils/SQLDBUtils.java

```diff
@@ -0,0 +1,245 @@
+     *
+     * @param jdbcURL  JDBC URL
+     * @param username Username for the DB
+     * @param password Password to connect to the DB
+     * @param sqlFile  SQL statements for initialization.
+     */
+    public static void initDatabase(String jdbcURL, String username, String password, String sqlFile) {
+        try (Connection connection = DriverManager.getConnection(jdbcURL, username, password);
+             Statement st = connection.createStatement()) {
+            String sql = readFileToString(sqlFile);
+            String[] sqlQuery = sql.trim().split("/");
+            for (String query : sqlQuery) {
+                st.executeUpdate(query.trim());
+            }
+            if (!connection.getAutoCommit()) {
+                connection.commit();
+            }
+        } catch (SQLException e) {
+            log.error("Error while initializing database: ", e);
+        }
+    }
+
+    /**
+     * Delete the given directory along with all files and sub directories.
+     *
+     * @param directory Directory to delete.
+     */
+    public static boolean deleteDirectory(File directory) {
+        if (!directory.isDirectory()) {
+            return directory.delete();
+        }
+        for (File f : directory.listFiles()) {
+            boolean success = deleteDirectory(f);
+            if (!success) {
+                return false;
+            }
+        }
+        return true;
+    }
+
+    /**
+     * Delete all the files and sub directories which matches given prefix in a given directory.
+     *
+     * @param directory Directory which contains files to delete.
+     * @param affix     Affix for finding the matching files to delete.
+     */
+    public static void deleteFiles(File directory, String affix) {
+        if (!directory.isDirectory()) {
+            return;
+        }
+
+        for (File f : directory.listFiles()) {
+            if (f.getName().startsWith(affix) || f.getName().endsWith(affix)) {
+                deleteDirectory(f);
+            }
+        }
+    }
+
+    private static String readFileToString(String path) {
+        InputStream is = null;
+        String fileAsString = null;
+        URL fileResource = BCompileUtil.class.getClassLoader().getResource(path);
+        try {
+            is = new FileInputStream(fileResource.getFile());
+            BufferedReader buf = new BufferedReader(new InputStreamReader(is));
+            String line = buf.readLine();
+            StringBuilder sb = new StringBuilder();
+            while (line != null) {
+                sb.append(line).append("\n");
+                line = buf.readLine();
+            }
+            fileAsString = sb.toString();
+        } catch (IOException e) {
+            // Ignore here
+        }
+        return fileAsString;
+    }
+
+    /**
+     * This class represents a database used for testing data clients.
+     */
+    public abstract static class TestDatabase {
+
+        String jdbcUrl;
```

Comment:
Don't we need to make these fields private?

Reply:
These field are used in the other classes as well. Cannot make them private.